packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -283,9 +309,42 @@ const ReleasesPage = () => {
   };
 
   return (
-    <ReleasesLayout onClickAddRelease={toggleAddReleaseModal} totalReleases={totalReleases}>
+    <ReleasesLayout
+      onClickAddRelease={toggleAddReleaseModal}
+      totalReleases={totalReleases}
+      isPrimaryActionDisabled={hasReachedMaximumPendingReleases}
+    >
       <ContentLayout>
         <>
+          {activeTab === 'pending' && hasReachedMaximumPendingReleases && (
+            <CustomAlert
+              marginTop={6}
```

Comment:
I don't think the designs have this marginTop, only marginBottom

Reply:
To be honest I think we need the marginTop by design
here you can see the solution without marginTop
<img width="1511" alt="withoutMarginTop" src="https://github.com/strapi/strapi/assets/2589748/a90e5a64-8b5a-4960-85d4-bf0d6c5052f2">
and here you can see the spaces by design
<img width="810" alt="figma" src="https://github.com/strapi/strapi/assets/2589748/b9b10ad1-c053-4b19-939a-27e86b40bf0a">

Reply:
I think there is another issue at play here. The way I interpret the designs there is 40px marginBottom in the Header. The distance from 3 releases to the banner displaying the release limit should be that marginBottom, 40px. So there should be no need for marginTop except for some reason the banner is partially inside the Header. I think we should investigate why that is. Applying marginTop feels like covering up an issue. 

![Screenshot 2024-01-26 at 17 57 12](https://github.com/strapi/strapi/assets/26598053/5eac1cd7-416b-4eb1-ae86-810fd1a9dbbf)

Reply:
Found the bug, inside the HeaderLayout component we have a height for the wrapper calculated based on the content, https://github.com/strapi/design-system/blob/ab8326385e52663ec898814b30d8beeda37b1659/packages/strapi-design-system/src/Layout/HeaderLayout.tsx#L46, in our case the height was calculated with all the content without subtitle because it is filled with content coming from an API. To avoid this I have now added a default subtitle content "No releases" when we are waiting for the API response